packages/subgraph/tests/pool-metadata-registry.test.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  assert,
+  describe,
+  test,
+  clearStore,
+  beforeAll,
+  afterAll
+} from "matchstick-as/assembly/index"
+import { Bytes } from "@graphprotocol/graph-ts"
+import { PoolMetadataUpdated } from "../generated/schema"
+import { PoolMetadataUpdated as PoolMetadataUpdatedEvent } from "../generated/PoolMetadataRegistry/PoolMetadataRegistry"
+import { handlePoolMetadataUpdated } from "../src/pool-metadata-registry"
+import { createPoolMetadataUpdatedEvent } from "./pool-metadata-registry-utils"
+
+// Tests structure (matchstick-as >=0.5.0)
+// https://thegraph.com/docs/en/developer/matchstick/#tests-structure-0-5-0
+
+describe("Describe entity assertions", () => {
+  beforeAll(() => {
+    let poolId = Bytes.fromI32(1234567890)
+    let metadataCID = "Example string value"
```

Comment:
this should look like a IPFS CID e.g. `QmPq9...`

Reply:
Sure... I'll be fixing it on the next [issue](https://linear.app/bleu-llc/issue/BAL-36/create-mapping-for-handling-metadata-updates-events)